pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -157,37 +221,43 @@ func (r *reconciler) reconcileIPServiceEntry(ctx context.Context, ipHosts []dtcl
 	return nil
 }
 
+func (r *reconciler) cleanupIPServiceEntry(ctx context.Context, component string) error {
+	entryName := BuildNameForIPServiceEntry(r.client.Owner.GetName(), component)
+
+	err := r.client.DeleteServiceEntry(ctx, entryName)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
this code can be simplified, you are doing nothing with err. so
```
return r.client.DeleteServiceEntry(ctx, entryName)
```

Reply:
fixed in a6858713